lib/index.js

```diff
@@ -48,30 +48,13 @@ SatisMeter.prototype.loaded = function() {
  * @param {Identify} identify
  */
 
-SatisMeter.prototype.identify = function(identify) {
-  var traits = identify.traits();
-  traits.token = this.options.apiKey || this.options.token;
-  traits.user = {
-    id: identify.userId()
-  };
-
-  if (identify.name()) {
-    traits.user.name = identify.name();
-  }
-  if (identify.email()) {
-    traits.user.email = identify.email();
-  }
-  if (identify.created()) {
-    traits.user.signUpDate = identify.created().toISOString();
-  }
-
-  // Remove traits that are already passed in user object
-  delete traits.id;
-  delete traits.email;
-  delete traits.name;
-  delete traits.created;
-
-  window.satismeter(traits);
+SatisMeter.prototype.identify = function() {
+  window.satismeter({
+    writeKey: this.options.apiKey || this.options.token,
+    userId: this.analytics.user().id(),
+    traits: this.analytics.user().traits(),
```

Comment:
ahh yeah that behavior is legacy... I don't like it either. 

So i guess we can just leave this as is. otherwise you'd have to manually delete `id` from the the returning object after `.traits()`